softwareconfiguration/model/src/main/java/ai/libs/softwareconfiguration/model/NumericParameterDomain.java

```diff
@@ -0,0 +1,29 @@
+package ai.libs.softwareconfiguration.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import ai.libs.jaicore.basic.sets.Interval;
+
+public class NumericParameterDomain extends Interval implements IParameterDomain {
```

Comment:
Still not sure whether the isInteger flag is better than having an inheriting class IntegerParameterDomain...

Reply:
maybe, but this currently works, and there is no concrete reason for now to change it.